src/domains/reminder/services/use-create-one.js

```diff
@@ -0,0 +1,25 @@
+// import useSWR from "swr";
+import fetch from "unfetch";
+
+
+const useCreateReminder = async(reminder) => {
+	const response = await fetch("http://localhost:3001/reminder", {
+		method: "POST",
+		headers: { "Content-Type":"application/json" },
+		body: JSON.stringify(reminder)
```

Comment:
Ca devrait fonctionner sans le JSON.stringify je pense

Reply:
Je n'ai pas réussi à corriger. J'ai une erreur 400 et le message ci-dessous dans la console:
Access to XMLHttpRequest at 'http://localhost:3001/reminder' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
Je veux bien qu'on en discute parce que pour moi CORS c'était allowed et en ce qui concerne le header je suis un peu pommée.
Faut-il faire une modif dans le fichier app.config du back (cors) ?